param/__init__.py

```diff
@@ -1164,8 +1164,9 @@ class ObjectSelector(SelectorBase):
 
     # ObjectSelector is usually used to allow selection from a list of
     # existing objects, therefore instantiate is False by default.
-    def __init__(self,default=None,objects=None,instantiate=False,
-                 compute_default_fn=None,check_on_set=None,allow_None=None,**params):
+    def __init__(self, default=None, objects=None, instantiate=False,
+                 compute_default_fn=None, check_on_set=None, allow_None=False,
```

Comment:
Team: looks like https://github.com/holoviz/param/issues/106, from 2015

Reply:
Sure, but i see no comment in #106 that would let me know if there was a reason why default value for `allow_None` is None. I do not see any reason and consider it an error, and i wanted to change it to be able to compute the number of possible values as `param.allow_None + len(param.objects)`, instead of having to write `bool(param.allow_None) + len(param.objects)`.

Now i see that the default value needs also to be changed in `Selector.__init__`, i add a commit for this.